mne/io/bti/bti.py

```diff
@@ -926,10 +927,13 @@ def _read_bti_header(pdf_fname, config_fname, sort_by_ch_name=True):
             ch['loc'] = _coil_trans_to_loc(ch_cfg['dev']['transform'])
         else:
             ch['loc'] = None
-        if info['data_format'] <= 2:  # see DTYPES, implies integer
-            ch['cal'] = ch['scale'] * ch['upb'] / float(ch['gain'])
-        else:  # float
-            ch['cal'] = ch['scale'] * ch['gain']
+        if pdf_fname is not None:
+            if info['data_format'] <= 2:  # see DTYPES, implies integer
+                ch['cal'] = ch['scale'] * ch['upb'] / float(ch['gain'])
+            else:  # float
+                ch['cal'] = ch['scale'] * ch['gain']
+        else:
+            ch['scale'] = 1.0
```

Comment:
It's confusing why in one case `cal` gets set but not `scale`, and `scale` gets set in the other place and not `cal`. Can you add a comment saying why, e.g.:
```
# the defaults for cal and scale work, so we only need to set one or the other here:
```
Because I don't expect that to be the case but maybe it is.